django_password_validators/password_history/password_validation.py

```diff
@@ -12,11 +12,22 @@
 
 class UniquePasswordsValidator(object):
     """
-    Validate whether the password was once used by the user.
-
+    Validate whether the password was once used by the user
+    in a determined range.
     The password is only checked for an existing user.
     """
 
+    def __init__(self, lookup_range=float('inf')):
+        try:
+            self.lookup_range = lookup_range if lookup_range >= 0 else float(
+                'inf')
+        except BaseException:
+            raise TypeError(
+                'UniquePasswordsValidator Error: lookup_range is not a positive integer')
+        self.validation_error = ValidationError(
+            _("You cannot use a password that was recently used in this application."),
+            code='password_used')
```

Comment:
@spokerman12, if you want to avoid exceeding the 79-charracters width, you can do something like

```suggestion
        except BaseException:                                                                                                                                                                                              
            raise TypeError(                                                                                                                                                                                               
                'UniquePasswordsValidator Error: lookup_range is not a '                                                                                                                                                   
                'positive integer'                                                                                                                                                                                         
            )                                                                                                                                                                                                              
        self.validation_error = ValidationError(                                                                                                                                                                           
            _(                                                                                                                                                                                                             
                "You cannot use a password that was recently used "                                                                                                                                                        
                "in this application."                                                                                                                                                                                     
            ),                                                                                                                                                                                                             
            code='password_used'                                                                                                                                                                                           
        )                                                                                                                                                                                                                  
```
which is a lot more readable, no? Also, leaving the closing brackets in a separate line looks a lot more readable and clean imho.